migration_original/ODS1Stage/tables/Base/ClientProductToPartner/spu_translated_ClientProductToPartner.sql

```diff
@@ -190,7 +190,7 @@ insert_statement := $$
                          )
                     values 
                         (
-                        uuid_string(),                        
+                        utils.generate_uuid(source.clienttoproductid || source.partnerid || source.sourcecode), 
```

Comment:
are you sure sourcecode should be here? don't think so

Reply:
My bad, it shouldn't be there, I will remove it